src/System.Net.Http/tests/FunctionalTests/Configurations.props

```diff
@@ -6,6 +6,7 @@
       netcoreapp-Unix;
       netstandard-Windows_NT;
       netstandard-Unix;
+      uap-Windows_NT;
```

Comment:
Not strictly necessary, but taking the change as is seems cleaner and more traceable. Given that the change is trivial that seems the better choice to me.